packages/raystack/avatar/avatar.module.css

```diff
@@ -209,15 +314,26 @@
   position: relative;
 }
 
-.avatarWrapper:hover {
-  z-index: 2;
-}
-
 .avatarWrapper:last-child {
   z-index: 1;
 }
 
 .avatarGroup .avatar {
-  border: 1px solid var(--background-base, white);
+  border: 1px solid var(--rs-background-base-primary);
   box-sizing: content-box;
-}
\ No newline at end of file
+}
+
+/* Avatar Group Size Variants */
+.avatar-size-1 .avatarWrapper { --avatar-overlap: 0.8em; }
```

Comment:
@rsbh @anujk14 This defines the overlap extent on each size level. I need to optimise this part more as exact overlap values were not mentioned in Figma.